openarena-q3e

```diff
@@ -1,29 +1,61 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
+# Check if the config file exists
+if [ -f "$CONFIG_FILE" ]; then
+    source "$CONFIG_FILE"
+fi
+# Check if it's the first run or if RESPECT_XDGDIRS is not set
+if [ ! -f "$CONFIG_FILE" ] || [ -z "$RESPECT_XDGDIRS" ]; then
+    RESPECT_XDGDIRS=0
+
+    # Save RESPECT_XDGDIRS in the config file
+    echo "RESPECT_XDGDIRS=$RESPECT_XDGDIRS" > "$CONFIG_FILE"
+fi
+# Use the value of RESPECT_XDGDIRS to determine the directory
+if [ "$RESPECT_XDGDIRS" -eq 1 ]; then
+    # Use ${XDG_DATA_HOME}/homepath directory
+    mkdir -p "${XDG_DATA_HOME}/homepath"
```

Comment:
I would remove the `/homepath` subdirectory and just use `${XDG_DATA_HOME}`.

Reply:
I'd like to make it clear what the files on the folder are.

Reply:
By making a directory called "homepath", you won't make it clear for end-users, you will make it confusing. At least choose a different name.